accessman.go

```diff
@@ -125,35 +160,53 @@ func (a *accessMan) newPendingOpenChan(remotePub *btcec.PublicKey) error {
 	a.banScoreMtx.Lock()
 	defer a.banScoreMtx.Unlock()
 
+	acsmLog.DebugS(ctxb, "Processing new pending open channel",
```

Comment:
There's one too many or too few attributes. `"peer"` doesn't have a value.

Reply:
yeah, the `btclog.Hex(...)` inserts both a key _and_ a value. So you'd probably for this just want:
`btclog.Hex("peer", remotePub.SerializeCompressed())`

Reply:
Ah ok that makes sense, fixed!